### ISSUE TITLE
Low psnr on the validation set (Adobe 240 dataset)

### ISSUE THREAD

**Issue by Superlee506** [avinashpaliwal/Super-SloMo#62]
I followed all the preprocessed steps, and load the pretrained model you provived, when I fine-tune this model, I find that the ValPSNR is really lower than the paper which is 31.19.
Any suggestion would be appreciated.

"Loss: 20.166218  Iterations:    0/1383  TrainExecTime: 1.0  ValLoss:18.554750  ValPSNR: 24.7886  ValEvalTime: 12.00 LearningRate: 0.000100
 Loss: 14.172664  Iterations:    1/1383  TrainExecTime: 1.1  ValLoss:19.047856  ValPSNR: 24.7444  ValEvalTime: 12.15 LearningRate: 0.000100
 Loss: 19.894939  Iterations:    2/1383  TrainExecTime: 1.1  ValLoss:19.435021  ValPSNR: 24.7141  ValEvalTime: 11.92 LearningRate: 0.000100
 Loss: 16.571831  Iterations:    3/1383  TrainExecTime: 1.1  ValLoss:19.684721  ValPSNR: 24.7064  ValEvalTime: 11.95 LearningRate: 0.000100
 Loss: 11.380559  Iterations:    4/1383  TrainExecTime: 1.1  ValLoss:19.803742  ValPSNR: 24.7096  ValEvalTime: 12.03 LearningRate: 0.000100
 Loss: 9.545274  Iterations:    5/1383  TrainExecTime: 1.1  ValLoss:19.904443  ValPSNR: 24.7032  ValEvalTime: 11.95 LearningRate: 0.000100
 Loss: 17.877144  Iterations:    6/1383  TrainExecTime: 1.1  ValLoss:19.801204  ValPSNR: 24.7159  ValEvalTime: 11.84 LearningRate: 0.000100
 Loss: 22.639399  Iterations:    7/1383  TrainExecTime: 1.1  ValLoss:19.594337  ValPSNR: 24.7272  ValEvalTime: 11.82 LearningRate: 0.000100
 Loss: 13.224724  Iterations:    8/1383  TrainExecTime: 1.1  ValLoss:19.394606  ValPSNR: 24.7286  ValEvalTime: 11.86 LearningRate: 0.000100
 Loss: 13.820227  Iterations:    9/1383  TrainExecTime: 1.1  ValLoss:19.320602  ValPSNR: 24.7373  ValEvalTime: 11.87 LearningRate: 0.000100
 Loss: 7.078848  Iterations:   10/1383  TrainExecTime: 1.1  ValLoss:19.499358  ValPSNR: 24.7295  ValEvalTime: 11.84 LearningRate: 0.000100"

**Comment by Superlee506** [comment]
@avinashpaliwal

**Comment by avinashpaliwal** [comment]
Hi @Superlee506. The results in the paper and the validation set psnr are two different things. The validation set is a very small sample set to observe the training performance of the model. Although it is weird that pretrained model starts with such a low psnr. Can you make sure that you are loading the checkpoint correctly. Is the flag set to true and the checkpoint path correct?
